content/en/tracing/llm_observability/tracing_concepts.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Tracing Concepts
+---
+
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+## Spans
+
+A span is a unit of work representing an operation in your LLM application, and is the building block of a trace.
+
+### Span Attributes
+
+A span consists of the following attributes:
+- Name of the traced operation
+- Operation timestamp data
+- Error information including type, message, and traceback
+- Inputs and Outputs to the traced operation (e.g. LLM prompts and completions)
+- Metadata relevant to the traced operation (e.g. LLM parameters such as `temperature`, `max_tokens`)
+- Metrics relevant to the traced operation (e.g. `prompt_tokens`, `completion_tokens`)
+- Tags
+
+### Span Kinds
+
+LLM Observability categorizes spans by their span kind, which defines the type of work the span is performing. This can give you more granular insights on what operations are being performed by your LLM application. LLM Observability currently supports the following span kinds:
+- *LLM*: A call to an LLM.
+- *Workflow*: A sequence of operations which include LLM calls and any supporting operations.
+- *Agent*: A series of decisions and operations made by an autonomous agent.
+- *Tool*: A call to external programs or services, such as a web API or a database.
+- *Task*: Any standalone step in a workflow or agent that does not involve a call to an external service.
+- *Embedding*: A call to an embedding model or function.
+- *Retrieval*: A data retrieval operation from an external knowledge base.
+
+To learn more about each span kind, see the [Span kind][1] page.
+
+
+## Traces
+
+A trace represents the work involved in processing a request in your LLM application, and consists of one or more nested spans. A *root span* is the first span in a trace, and marks the beginning and end of the trace.
+
+Datadog’s LLM Observability product is designed to support observability for a variety of LLM applications, from simple to complex:
+
+### LLM Inference Monitoring
```

Comment:
I think we should number these three sections e.g. 

### 1. LLM Inference Monitoring

to make it clear that we're going in order